grails-app/views/userProfile/index.gsp

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+	    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+	    <r:require module="main_mod"/>
+	    <title>User profile</title>
+	    <r:layoutResources/>
```

Comment:
use a layout - no need to copy/paste stuff like this into every GSP. then making changes can happen in one place